frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -56,6 +56,9 @@ namespace ExtensionProperties {
 
     /** Loader for the corresponding dashboard card component. */
     loader: LazyLoader<any>;
+
+    /** The card span in grid. Property is ignored for small screen and defaults to 12. */
```

Comment:
Nit: `Card's vertical span in the column. Ignored for small screens, defaults to 12.`

Reply:
fixed